### PR TITLE
Add example import for package level types

### PR DESCRIPTION
**Pull request by jgresham**
I couldn't find an example import for package level types documented elsewhere

## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ You can use `web3.js` as follows:
 
 ```typescript
 import Web3 from 'web3';
+import { BlockHeader, Block } from 'web3-eth' // ex. package types
```
Comment (nazarhussain) [inline README.md:92]: I don't understand the reason or benefit of changing this example. If you think an example for other packages is good to have, then would suggest to create a separate example for it.

**Review by nazarhussain — CHANGES_REQUESTED** [review]

**Comment by nazarhussain** [comment]
Hi @jgresham Thanks for contribution. I posted some feedback. In addition to it I suggest you to create an issue on Github repo and link your PR to it. Also create an entry to `CHANGELOG.md` specifying your PR.

**Comment by jgresham** [comment]
@nazarhussain When I wanted to import the `Block` type from web3, I did not know where to import them from. I intuitively thought it would be `import { Block } from 'web3'` but that was not the case and I had to search web3 source code to find the import. Perhaps I am missing a tool which would have automatically found that it is `import { Block } from 'web3-eth'`.. but if this is not a common tool that users have installed, then I think it would benefit users to have this documented somewhere. If needed, where would you suggest it be documented?

**Comment by nazarhussain** [comment]
@jgresham As we are focused on `4.x` rewrite which is done completely in Typescript. So you will be get more safe and more consistent developer experience with that release. For `1.x` seems mentioning it the README is fine.

**Review by nazarhussain — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Pull request merged**